components/Molecule/contact-form.html

```diff
@@ -0,0 +1,53 @@
+<div class="component"  title="Contact Form" version="1">
+
+  <style>
+
+form{
+      display: grid;
+      max-width: 50%;
+      margin: 100px 180px 0px;
+    }
+
+.contact-form label{
+      color: #747474;
+    }
+.contact-form input{
+      line-height: 1.5rem;
+      margin-bottom: 20px;
+
+    }
+.contact-form textarea{
+      padding-bottom: 120px;
+    }
+
+.form-button{
+      max-width: 110px;
+      margin-top: 20px;
+      padding: 15px;
+      background-color: #1ab2cf;
+      color: #ffffff;
+      font-size: 1em;
+      font-weight: bold;
+      text-transform: uppercase;
+    }
+.form-button:hover{
+      background-color: #009dbb;
+    }
+  </style>
+    <markup>
```

Comment:
I'm not familiar with the `<markup>` tag. What is this used for?

Reply:
@Harrison,thank you. Honestly,  I refer to the example, which is in our project, and I understand it used to separate the component one from the other.

Reply:
@hwstovall, see https://github.com/team-refcode/refcode.org/pull/134 for more info on that. 

To simplify the pattern library code I used invented HTML tags. They won't find their way into the final markup so we're ok.